gateway/ln-gateway/src/bin/cln_extension.rs

```diff
@@ -638,7 +638,7 @@ impl GatewayLightning for ClnRpcService {
             .await
             .map_err(|e| Status::internal(e.to_string()))?
             .call(cln_rpc::Request::NewAddr(model::requests::NewaddrRequest {
-                addresstype: Some(cln_rpc::model::requests::NewaddrAddresstype::BECH32),
+                addresstype: None,
```

Comment:
This is the only change here that isn't dev-only. Is this OK? @tvolk131

Reply:
I changed `addresstype` to `None` to match the existing behavior in `open_channel()`: https://github.com/fedimint/fedimint/blob/3617b966081793198b46e56443fa2f78f506788a/devimint/src/external.rs#L575

I don't think this will likely matter, but I wanted to make sure that the implementations of `open_channel()` and `open_channel_between_gateways()` match as closely as possible.